### PR TITLE
[LANGUAGE] Update Czech (UBL and BLTouch)

### PR DESCRIPTION
**Pull request by petrzjunior**
Uff, finally completed the UBL set of strings.

### CODE REVIEW

**Comment by Roxy-3D** [comment]
Thank You!

**Comment by Roxy-3D** [comment]
Messages should not be too disruptive to merge with regard to the code freeze, right???

**Comment by petrzjunior** [comment]
>Messages should not be too disruptive with regard to the code freeze, right???

Don't understand, what you meant by that.

**Comment by thinkyhead** [comment]
> Messages should not be too disruptive

All fixes to obvious bugs, language updates, and merges of ready and tested PRs are fine and won't disrupt the code reorganization, which I will only be making the first preliminary run at this weekend.

The coming week we should aim to patch all known issues and next weekend I will aim to produce the finished 2.0.x branch based on developer commentary on my first attempt. That first attempt will be built tomorrow, starting from the known-working HAL branch. I will first make sure I comprehend all the changes, then I will keep HAL-stuff in mind as I do the re-organization.

I'll post more specifics over on the Marlin 2.0 issue tomorrow, when I have the branch posted.

File: Marlin/language_cz.h [hunk 1]
```diff
@@ -147,6 +213,7 @@
 #define MSG_NO_CARD                         _UxGT("Zadna SD karta")
 #define MSG_DWELL                           _UxGT("Uspano...")
 #define MSG_USERWAIT                        _UxGT("Cekani na uziv...")
+#define MSG_PRINT_PAUSED                          _UxGT("Tisk pozastaven")
```
Comment (thinkyhead) [inline Marlin/language_cz.h]: spacing

File: Marlin/language_cz.h [hunk 2]
```diff
@@ -77,6 +78,68 @@
...
 #define MSG_BED_LEVELING                    _UxGT("Vyrovnat podlozku")
 #define MSG_LEVEL_BED                       _UxGT("Vyrovnat podlozku")
+#define MSG_EDITING_STOPPED                 _UxGT("Konec uprav site")
+
+#define MSG_UBL_DOING_G29                   _UxGT("Provadim G29")
+#define MSG_UBL_UNHOMED                     _UxGT("Prejedte domu")
+#define MSG_UBL_TOOLS                       _UxGT("UBL nastroje")
+#define MSG_UBL_LEVEL_BED                   _UxGT("Unified Bed Leveling")
+#define MSG_UBL_MANUAL_MESH                 _UxGT("Manualni sit bodu")
+#define MSG_UBL_BC_INSERT                   _UxGT("Polozte kreditku a zmerte")
```
Comment (thinkyhead) [inline Marlin/language_cz.h]: You can test for over-long strings using a "ruler". These strings are all too long:
```
12345678901234567890
Polozte kreditku a z|merte
Odstrnte a zmerte po|dlozku
Zkontrolovat vlastni| sit
```

Comment (petrzjunior) [reply Marlin/language_cz.h]: Yeah, I know that they are long, but I've seen >18 string in `language_en.h` as well. All seem to fit in 20 characters, so is 20 new rule of thumb?

**Comment by Roxy-3D** [comment]
>> Messages should not be too disruptive with regard to the code freeze, right???

> Don't understand, what you meant by that.

My apologies @petrzjunior     I thought it was clear the message was intended for Thinky.   Thinky needs the code frozen right now to move it to the new folder and file layout.  My comment was more to the idea that "Messages can change without disrupting things, right?"

**Comment by Roxy-3D** [comment]
> That first attempt will be built tomorrow, starting from the known-working HAL branch. I will first make sure I comprehend all the changes, then I will keep HAL-stuff in mind as I do the re-organization.

@thinkyhead  Right now the 'Known working HAL branch' only handles the `REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER`. It would be really nice if it could also compile in the `REPRAP_DISCOUNT_SMART_CONTROLLER`. I don't think it necessarily needs to compile with all the different LCD Panels. But these two would be a really good starting place.

**Comment by thinkyhead** [comment]
Ready to merge at #7153 — thank you!

**Pull request closed without merging**